src/purifier/package.json

```diff
@@ -1,15 +1,23 @@
 {
   "name": "@ampproject/purifier",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "description": "AMP-specific sanitization library",
   "author": "The AMP HTML Authors",
   "license": "Apache-2.0",
-  "main": "dist/purifier.js",
-  "files": ["dist/*"],
+  "main": "dist/purifier.mjs",
```

Comment:
Is only having an "mjs" output typical?

Reply:
My impression was that `main: ` should be CJS and `module: ` is normally mjs.

Reply:
Nice, didn't know this. Added CJS output now and used `main` and `module` in package.json.

Reply:
Looks like this was a bridge until Node.js supported ES6 modules: https://nodejs.org/api/esm.html#esm_dual_commonjs_es_module_packages

Reply:
Reading a bit into this, it seems the `module` + `main` approach @samouri suggested is still best, since Node.js will just ignore `module` and use `main`. The solution from nodejs.org to have a wrapper instead seems only useful to allow ESM support in Node (which is still experimental in LTS) and I fear it will cause complications when used in a browser.

TLDR: I see no real advantages in having an ESM version in Node (for now).

Reply:
Agreed, just sharing for anyone who was also curious.

Reply:
Ah, got it, sorry I misunderstood your previous comment.